src/v2/view-builder/views/ov/ChallengeOktaVerifyTotpView.js

```diff
@@ -13,8 +15,40 @@ const Body = BaseForm.extend(Object.assign(
     save() {
       return loc('mfa.challenge.verify', 'login');
     },
+
+    initialize: function() {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      this.listenTo(this.options.appState, 'afterError', (convertedErrors) => {
+        // Need to wait until BaseForm renders the error so we can replace it
+        setTimeout(() => {
```

Comment:
can you try to avoid `setTimeout` here ?

Reply:
any suggestions? I need to form to go its regular flow and render the error message, so I can replace it. I have no access to the exact part where the BaseForm renders that message, so this seems to me a good alternative

Reply:
for see how `IdentifierView.js` overrides `showMessages()`